CMS/static/jsonfiles/institutions.json

```diff
@@ -1218,8 +1218,8 @@
         },
         {
             "alphabet": "u",
-            "name": "UWE",
-            "order_by_name": "uwe"
+            "name": "Uwe Bristol",
```

Comment:
Should this not be UWE Bristol?

Reply:
lol - new exception in the database - good spot - that means im going to have to update the backend service to handle another type and require another data load. Awesome!

Reply:
Very good spot though

Reply:
Sorry